iam-policy-autopilot-policy-generation/build.rs

```diff
@@ -49,6 +65,17 @@ struct ShapeReference {
     shape: String,
 }
 
+// Git version and commit hash for boto3 and botocore
+#[derive(Debug, Clone, Serialize, Deserialize)]
+struct GitSubmoduleVersion {
+    #[serde(rename = "gitCommit")]
+    git_commit_hash: String,
+    #[serde(rename = "gitTag")]
+    git_tag: Option<String>,
+    #[serde(rename = "dataHash")]
+    data_hash: String,
```

Comment:
Could you add a code comment about why we need commit hash, tag, and data hash? Could we get the commit from the tag during comparison? We should only need to update on a new tag, so I don't think we need to include all of this information here.

I'm particularly wondering why we need to include `data_hash` here. We could just compute that hash as part of the comparison between current build and latest submodule, as long as we know what submodule commit we used to build the current version.